source/base/Profiler.ooc

```diff
@@ -50,6 +50,19 @@ Profiler: class {
 		This _profilers free()
 		This _profilers = null
 	}
+	benchmark: static func (task: Func, timeoutMilliseconds := 1000.0) -> Double {
```

Comment:
I'm not sure what's appropriate, but maybe we should pass and return `TimeSpan`s instead?